### ISSUE TITLE
Cannot test focus ring in fast/forms/input-text-click-{inside,outside}.html

### ISSUE THREAD

**Issue by annulen** [annulen/webkit#156]
Probably missing DRT feature

**Comment by annulen** [comment]
Also
fast/forms/textfield-focus-ring

**Comment by annulen** [comment]
fast/forms/textfield-overflow

**Comment by annulen** [comment]
fast/transforms/transformed-focused-text-input.html
